pkg/vmimage/vmimage.go

```diff
@@ -252,6 +253,13 @@ func (builder *Builder) Run(ctx context.Context) error {
 		return err
 	}
 
+	go builder.sshInstall()
+	// wait for install
+	err = builder.waitForInstall(ctx)
```

Comment:
code suggests this is to wait until the install _begins_, is that correct? I am thinking you should need a /var/log/install.done or similar to check when the install is finished, touch it at the end of the install script ?

Reply:
When the `install.sh` finishes it writes here: https://github.com/openshift/openshift-azure/pull/1905/files#diff-037e2465ca7d49e6da64f82bd06ca8e1R349

This was the intention.  There might be a better way